combine-events/index.d.ts

```diff
@@ -10,3 +10,15 @@ export function combineEvents<Shape>(
 ): Event<
   { [K in keyof Shape]: Shape[K] extends Event<infer U> ? U : Shape[K] }
 >;
+export function combineEvents<Shape extends Tuple>(
+  shape: Shape,
+  reset: Unit<any>,
```

Comment:
I think it is better to add configuration.

`{ reset }`

Reply:
And also the object form:

```ts
combineEvents({
  shape: {...},
  reset: unit,
})
```

Reply:
How are you going to differentiate if the first object is `source` or `config`?

Reply:
I like the idea to use `combineEvents(config)`

```js
combineEvents({
  source: {...},
  reset: unit, 
  target,
})
```

No more signatures needed, config only.